README.md

```diff
@@ -21,11 +21,22 @@ another.
 - Make a PR and explain what the package is for
 
 ## Building
-- Generate a valid anaconda token, preferrably one that will expire soon
-- Activate a conda environment with `conda-build=3.5.1` and `anaconda-client`
-  installed
-- `./build.py --channel pcds-tag --token $ANACONDA_TOKEN`
+`build.py` will slowly check all the package variants in the build matrix and
+build the variants that are missing on `pcds-tag`. Yes, we can make this faster
+with some effort.
```

Comment:
I actually did make this faster! But I guess it's technically still slow.

Reply:
Want me to take it out? Slow is always relative ...

Reply:
Nah, best to keep expectations low